### DISCUSSION TITLE
PixArt-LCM is available!!!

### DISCUSSION THREAD

**Discussion by lawrence-cj** [PixArt-alpha/PixArt-alpha#46] [Announcements]
PixArt-LCM is available:
demo: https://huggingface.co/spaces/PixArt-alpha/PixArt-LCM
code: https://github.com/PixArt-alpha/PixArt-alpha

We highly encourage users to use the fastest version of PixArt-LCM and give any valuable suggestions for us.

**Comment by Bigfield77** [comment]
It is super fast! :) I will test it more tonight

Great work!

**Comment by ardha27** [comment]
where i can download the LCM models?

**Reply by lawrence-cj** [reply]
Here it is: https://huggingface.co/PixArt-alpha/PixArt-LCM-XL-2-1024-MS/tree/main.
